src/DGtal/geometry/volumes/distance/VoronoiMap.h

```diff
@@ -229,6 +229,11 @@ namespace DGtal
      */
     ~VoronoiMap();
 
+    /**
+     * Disabling default constructor.
+     */
+     VoronoiMap() = delete;
```

Comment:
By the way, C++ doesn't implicitly declare the default constructor if any other constructor is defined. So deleting it is not mandatory even if it is clearer.